c/py_datatable.cc

```diff
@@ -261,6 +262,23 @@ PyObject* delete_columns(obj* self, PyObject* args) {
 
 
 
+PyObject* aggregate(obj* self, PyObject* args) {
```

Comment:
This method should not be in (py)DataTable class: the class will become user-facing once #1066 is implemented, so we want to keep the API clean.
The easiest approach is to declare this function global (e.g. listed in `DatatableModuleMethods` in  datatablemodule.c) and move its body to extras/aggregate.cc

Reply:
Ok, got it.

Reply:
Fixed.

Reply:
Fixed.